demo/istio/manifests/echo-service.yaml

```diff
@@ -1,3 +1,12 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: my-istio-namespace
```

Comment:
```suggestion
  name: demo
```

Reply:
you also need to add the namespace to the resources in the manifests

Reply:
@eddycharly done